migrations/025_create_customer_representatives.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE if not exists customer_representatives (
```

Comment:
nit: This might be a bit late in the game; but `representative` seems a bit wordy.

Would something like `agent` be preferrable?

Reply:
Thanks for the feedback, @vxio !  I agree with you that the word `representative` is a bit of a mouth full.  The wording was determined by product folks, and I know they've done a lot of the work on the UI using that term, so it may be a bit late to change it.

Reply:
I changed it to just `representative` based on yours and other feedback I received.  Hopefully that's not quite as much of a mouth full 😄

Reply:
lol nice!